### PR TITLE
Gyroscope integration and better control

### PR DESCRIPTION
**Pull request by Peque**

### CODE REVIEW

File: src/target/move.c [hunk 1]
```diff
@@ -138,11 +138,11 @@ static void wait_front_perpendicular(float error)
 	float average = 0.;
 
 	while (true) {
-		for (i = 0; i < 10; i++) {
```
Comment (cua-cua) [inline src/target/move.c]: SENSORS_CAL_SAMPLE_NUM? (Optional ;P)

File: src/control.c [hunk 2]
```diff
@@ -338,19 +338,17 @@ void motor_control(void)
 	left_speed = get_encoder_left_speed();
 	right_speed = get_encoder_right_speed();
 	encoder_feedback_linear = (left_speed + right_speed) / 2.;
-	encoder_feedback_angular = get_encoder_angular_speed();
+	encoder_feedback_angular = -get_gyro_z_radps();
```
Comment (cua-cua) [inline src/control.c]: Change the name of the variable. The angular feedback is not coming from the encoder now.

File: src/calibration.c [hunk 3]
```diff
@@ -153,6 +155,12 @@ void run_movement_sequence(const char *sequence)
 		case 'b':
 			turn_back();
 			break;
+		case 'k':
+			keep_front_wall_distance(CELL_DIMENSION / 2., 0.001);
```
Comment (cua-cua) [inline src/calibration.c]: 0.001?

Comment (Peque) [reply src/calibration.c]: 1 mm error allowed.

**Pull request merged**